src/software/ai/evaluation/calc_best_shot.h

```diff
@@ -51,5 +51,5 @@ std::optional<Shot> calcBestShotOnGoal(const Segment &goal_post, const Point &sh
 std::optional<Shot> calcBestShotOnGoal(const Field &field, const Team &friendly_team,
                                        const Team &enemy_team, const Point &shot_origin,
                                        TeamType goal,
-                                       double radius = ROBOT_MAX_RADIUS_METERS,
-                                       const std::vector<Robot> &robots_to_ignore = {});
+                                       const std::vector<Robot> &robots_to_ignore = {},
+                                       double radius = ROBOT_MAX_RADIUS_METERS);
```

Comment:
nit: comments need to be flipped as well

Reply:
Done.